packages/ensnode-sdk/src/ensindexer/config/types.ts

```diff
@@ -13,7 +13,7 @@ export enum PluginName {
   Lineanames = "lineanames",
   ThreeDNS = "threedns",
   ProtocolAcceleration = "protocol-acceleration",
-  Referrals = "referrals",
+  SubRegistry = "subregistry",
```

Comment:
```suggestion
  Subregistry = "subregistry",
```

The ENS community generally writes this as "Subregistry" and not "SubRegistry".